### ISSUE TITLE
Suggestion for gradient (non-issue)

### ISSUE THREAD

**Issue by Tijger-T** [noatpad/obsidian-banners#92]
It might be of interest to include the banner style (ie. Solid/Gradient) in the frontmatter settings. This way, the banner could be adapted to the page rather than relying on a global setting. The global setting could possibly be retained with the YAML providing an override function where used.

**Comment by MSBack** [comment]
Yeah, this would be great, because it really depends on the note if you want one or the other.

**Comment by termi** [comment]
It's possible. See https://github.com/noatpad/obsidian-banners/pull/147
![изображение](https://github.com/noatpad/obsidian-banners/assets/973643/f92c1c76-ffe1-468f-8dd8-7e8cc109217e)
